app/cdap/components/Administration/TetheringTabContent/ActionPopover/index.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import styled, { css } from 'styled-components';
+import Popover from 'components/shared/Popover';
```

Comment:
`components/shared/Popover` is built on `react-popper` and not Mui Popover. Could you use Mui `Popover` or `Dialog` for this component?

Reply:
@njbriggs I could, but the issue is adding the arrow to the top of the popover. Apparently only `Tooltip` and `Popper` support arrows inherently. `Tooltip` doesn't apply to our use case, and `Popper` with arrow is unnecessarily complicated compared to `react-popper` (same story with `Popover` in MUI). See [this](https://stackblitz.com/edit/poppers-with-arrows-x8a7ic?file=index.js) code sandbox for implementation. I can make it work if we've decided to stay away from react-popper but just wanted to make sure we're good with what MUI gives us.

Reply:
I put this together using MUI Popover. It's not 100% styled components but let me know what you think:

https://gist.github.com/mrahanjam/35c4e0819c14f72ea1d157dce750948d

Reply:
We can stick with `react-popper` then.